### PR TITLE
Remove CVC language support

### PR DESCRIPTION
**Pull request by mpreiner**
This commit removes the support for the CVC language and converts all *.cvc regression tests to SMT-LIBv2.

### CODE REVIEW

File: src/expr/dtype.cpp [hunk 1]
```diff
@@ -897,8 +897,6 @@ const std::vector<std::shared_ptr<DTypeConstructor> >& DType::getConstructors()
 
 std::ostream& operator<<(std::ostream& os, const DType& dt)
 {
-  // can only output datatypes in the cvc5 native language
-  language::SetLanguage::Scope ls(os, Language::LANG_CVC);
```
Comment (nafur) [inline src/expr/dtype.cpp:901]: I assume the comment was incorrect already?

Comment (mpreiner) [reply src/expr/dtype.cpp:901]: We can print datatypes in SMT-LIBv2. Not sure when this comment got obsolete. @ajreynol?

**Review by nafur — COMMENTED** [review]
I didn't look at the individual regressions, but the rest LGTM.
One general question: `checkEntailed` is apparently only used on tptp input now. We should consider getting rid of it entirely, or have some regressions that exercise it...

File: src/expr/dtype.cpp (hunk 1 above)
Comment (ajreynol) [reply src/expr/dtype.cpp:901]: This comment is extremely obsolete.

**Review by ajreynol — APPROVED** [review]
LGTM, this is great.

**Pull request merged**